### ISSUE TITLE
Bug: the example of cm-push is error 

### ISSUE THREAD

**Issue by dellkeji** [chartmuseum/helm-push#117]
## bug desc

the example of cm-push is error, the detail is the following

helm push xxx  xxx

should be

helm cm-push xxx xxx

```
helm cm-push -h

# output:
Helm plugin to push chart package to ChartMuseum

Examples:

  $ helm push mychart-0.1.0.tgz chartmuseum       # push .tgz from "helm package"
  $ helm push . chartmuseum                       # package and push chart directory
  $ helm push . --version="7c4d121" chartmuseum   # override version in Chart.yaml
  $ helm push . https://my.chart.repo.com         # push directly to chart repo URL

Usage:
  helm push [flags]

```

## version
helm version: 3.7.0
push plugin version: 0.10.0

**Comment by dellkeji** [comment]
@jdolitsky

**Comment by jdolitsky** [comment]
@dellkeji - thank you, this is now available in `v0.10.1`
